rdf-abac-core/src/main/java/io/telicent/jena/abac/attributes/syntax/tokens/TokenizerABAC.java

```diff
@@ -239,18 +211,18 @@ private Token parseToken() {
 
             // Multi-character symbols
             // Two character tokens : !=, GE >= , LE <=, &&, ||
-            case CH_EQUALS:     return maybeTwoChar(CH_EQUALS, '=', TokenType.EQ, "=", TokenType.EQUIVALENT, "==");
-            case CH_EMARK:      return maybeTwoChar(CH_EMARK, '=', TokenType.EMARK, "!", TokenType.NE, "!=");
-            case CH_LT:         return maybeTwoChar(CH_LT, '=', TokenType.LT, "<", TokenType.LE, "<=");
-            case CH_GT:         return maybeTwoChar(CH_GT, '=', TokenType.GT, ">", TokenType.GE, ">=");
+            case CH_EQUALS:     return maybeTwoChar(CH_EQUALS, CH_EQUALS, TokenType.EQ, TokenType.EQUIVALENT, "==");
+            case CH_EMARK:      return maybeTwoChar(CH_EMARK, CH_EQUALS, TokenType.EMARK, TokenType.NE, "!=");
+            case CH_LT:         return maybeTwoChar(CH_LT, CH_EQUALS, TokenType.LT, TokenType.LE, "<=");
+            case CH_GT:         return maybeTwoChar(CH_GT, CH_EQUALS, TokenType.GT, TokenType.GE, ">=");
 
-            case CH_VBAR:       return maybeTwoChar(CH_VBAR, '|', TokenType.VBAR, "|", TokenType.LOGICAL_OR, "||");
-            case CH_AMPHERSAND: return maybeTwoChar(CH_AMPHERSAND, '&', TokenType.AMPERSAND, "&", TokenType.LOGICAL_AND, "&&");
+            case CH_VBAR:       return maybeTwoChar(CH_VBAR, CH_VBAR, TokenType.VBAR, TokenType.LOGICAL_OR, "||");
+            case CH_AMPHERSAND: return maybeTwoChar(CH_AMPHERSAND, CH_AMPHERSAND, TokenType.AMPERSAND, TokenType.LOGICAL_AND, "&&");
```

Comment:
Random niggle - isn't `ampersand` misspelt here?  There shouldn't be a `h` in it should there?

Reply:
You're right! Unfortunately this comes from `jena-base` so out of our control here. I could raise a PR against Jena if you think it is likely to get approved?

Reply:
Probably not worth it, it's such a low level thing that most people aren't going to look at that code so doesn't really matter if the constant is misspelt